gateway-js/src/__tests__/testSetup.ts

```diff
@@ -1,4 +1 @@
-import './matchers/toMatchAST';
-import './matchers/toCallService';
-import './matchers/toHaveBeenCalledBefore';
-import './matchers/toHaveFetched';
+import 'apollo-federation-integration-testsuite';
```

Comment:
A bit less obvious what's happening here now, can we add a comment that says this loads the matchers?